x-pack/legacy/plugins/beats_management/public/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -11,6 +11,8 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { UIRoutes } from 'ui/routes';
 import { isLeft } from 'fp-ts/lib/Either';
+import { npSetup } from 'ui/new_platform';
```

Comment:
@mattapperson, I'm not sure why Beats team wasn't pinged by the GitHub automatically, but I see you were editing this file before, would you mind reviewing this single change in `beats_management` plugin?

Reply:
Pulling and checking now

Reply:
Hey @mattapperson, did you have chance to take a look at this change?